docs/readthedocs/models/input/grid/node.md

```diff
@@ -0,0 +1,62 @@
+(node-model)=
+
+# Node
+
+Representation of an electrical node, with no further distinction into bus bar, auxiliary node or others.
+
+## Attributes, Units and Remarks
+
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
+
+
+   * - Attribute
+     - Unit
+     - Remarks
+
+   * - uuid
+     - --
+     -
+
+   * - id
+     - --
+     - Human readable identifier
+
+   * - operator
+     - --
+     -
+
+   * - operationTime
+     - --
+     - Timely restriction of operation
+
+   * - vTarget
+     - p.u.
+     - Target voltage magnitude to be used by voltage regulation entities
+
+   * - slack
+     - --
+     - | Boolean indicator, if this nodes serves as a slack node in power
+       | flow calculation

```

Comment:
This is rendered as smaller text. Can you fix this?

Reply:
I noticed this before as well when I worked with list tables, don't know if it is easily fixable. Wouldn't spend too much time on this tbh, but you can still try if you like

Reply:
I haven't found anything concrete for the solution. It appears consistent, though, as it gets smaller with each line break.

Reply:
OK. I think we can keep these part as they are now.